src/components/search/SearchField.js

```diff
@@ -0,0 +1,121 @@
+import React,  { useState, useEffect } from 'react'
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Card } from 'antd';
+import useModal from '../Modal/useModal';
+import SearchModalContent from '../Modal/SeachModalFragment';
+import styled from "styled-components";
+import  data from '../search-bar/MOCK_DATA.json';
+
+const SearchField = ({ value }) => {
+    //state variables
+    const [ toggle, setToggle ] = useState(false);
+    const [ options, setOptions ] = useState();
+    const [ list, setList ] = useState([]);
+    //custom hooks
+    const SearchModal = Modal;
+    const {isVisible, selectedUser, toggleModal } = useModal(SearchModal)
+    
+    //styled components
+    const ListItem = styled.li `
+    width:100%;
+    &:hover {
+      background-color:rgba(0,0,0,.025);
+            cursor:pointer;
+     }
+    `;
+   //custom made filter funtion without any hard specificity currently, I'd like to make this better
+    const filterData = (value,options) => {
+        setList([{}])
+        const newList =  options.filter((val) => {
+            const fullName = `${val.first_name.toLowerCase()} ${val.last_name.toLowerCase()}`;
+            if(fullName.includes(value.name.toLowerCase())) {
+            return val
+        }
+        }).filter((val, idx) => idx < 10).map((val) => val) 
+                return(
+                    newList
+                )
+
+      }
+
+    //wrote out the api call originally going to local host and everything seemed to work , reverted to local mockdata file for development
+
+      useEffect(() => {
+      //   axios.get(`http://localhost:8000/mentees`)
```

Comment:
This would ultimately want to move to redux as an action. Might be worth setting up the plumbing for it and working from there